code/modules/mining/ore_redemption_machine/survey_vendor.dm

```diff
@@ -1,119 +1,94 @@
+	icon_vend = "exploration-vend"
 
 /obj/machinery/mineral/equipment_vendor/survey/Initialize(mapload)
 	. = ..()
-	//VOREStation Edit Start - Heavily modified list
 	prize_list = list()
 	prize_list["Gear"] = list(
-		EQUIPMENT("Defense Equipment - Smoke Bomb",				/obj/item/grenade/smokebomb,									10),
-		EQUIPMENT("Defense Equipment - Plasteel Machete",		/obj/item/material/knife/machete,							50),
-		EQUIPMENT("Defense Equipment - Razor Drone Deployer",	/obj/item/grenade/spawnergrenade/manhacks/station/locked,	100),
-		EQUIPMENT("Defense Equipment - Sentry Drone Deployer",	/obj/item/grenade/spawnergrenade/ward,						150),
-//		EQUIPMENT("Defense Equipment - Frontier Carbine",		/obj/item/gun/energy/locked/frontier/carbine,				750), //VOREStation removal
-		EQUIPMENT("Hybrid Equipment - Proto-Kinetic Dagger",	/obj/item/kinetic_crusher/machete/dagger,					75),
-		EQUIPMENT("Hybrid Equipment - Proto-Kinetic Machete",	/obj/item/kinetic_crusher/machete,							250),
-		EQUIPMENT("Fishing Net",								/obj/item/material/fishing_net,								50),
-		EQUIPMENT("Titanium Fishing Rod",						/obj/item/material/fishing_rod/modern,						100),
-		EQUIPMENT("Durasteel Fishing Rod",						/obj/item/material/fishing_rod/modern/strong,				750),
-		EQUIPMENT("Fulton Beacon",								/obj/item/fulton_core,												300),
-		EQUIPMENT("Geiger Counter",								/obj/item/geiger,											75),
-		EQUIPMENT("GPS Device",									/obj/item/gps/mining,										10),
-		EQUIPMENT("Jump Boots",									/obj/item/clothing/shoes/bhop,										250),
-		EQUIPMENT("Mini-Translocator",							/obj/item/perfect_tele/one_beacon,							120),
-		EQUIPMENT("Survey Tools - Mapping Unit",				/obj/item/mapping_unit,										150),
-		EQUIPMENT("Survey Tools - Mapping Beacon",				/obj/item/holomap_beacon,									25),
-		EQUIPMENT("Survival Equipment - Insulated Poncho",		/obj/random/thermalponcho,											75),
-		EQUIPMENT("Survival Equipment - Glowstick", 			/obj/item/flashlight/glowstick,								5),
-		EQUIPMENT("Survival Equipment - Flare", 				/obj/item/flashlight/flare,									5),
-		EQUIPMENT("Survival Equipment - Radioisotope Glowstick",		/obj/item/flashlight/glowstick/radioisotope,			40),
-		EQUIPMENT("Survival Equipment - Modular Explorer Suit",		/obj/item/clothing/suit/armor/pcarrier/explorer,				200),
-		EQUIPMENT("Survival Equipment - Armored Jumpsuit",		/obj/item/clothing/under/explorer/armored,							200),
+		EQUIPMENT("Brown Webbing",									/obj/item/clothing/accessory/storage/brown_vest,							500),
+		EQUIPMENT("Defense Equipment - Smoke Bomb",					/obj/item/grenade/smokebomb,												10),
+		EQUIPMENT("Defense Equipment - Razor Drone Deployer",		/obj/item/grenade/spawnergrenade/manhacks/station/locked,					100),
+		EQUIPMENT("Defense Equipment - Sentry Drone Deployer",		/obj/item/grenade/spawnergrenade/ward,										150),
+		EQUIPMENT("Defense Equipment - Frontier Carbine",			/obj/item/gun/energy/locked/frontier/carbine,								250),
```

Comment:
I think you accidentally uncommented the phaser carbine. Confusingly, the phasers and phase guns are different things despite the similar names.

Reply:
Ah, fugh. Let me find their phasegun replacement (if there is any?) and swap it with that.

Reply:
Probably /obj/item/gun/energy/phasegun or /obj/item/gun/energy/phasegun/rifle. The former is the same item size and the latter is maybe closer to a clear “upgrade”.

Reply:
Added it. Rifle seems to have a lot of shots so it gets bumped to 350. Carbine seems to be slightly more efficient than the pistol (but larger) and does more damage, so gave it 200.

I'm not 100% sure the value of things on the catalogurer, but this seems fine? If number adjustments are needed, someone can let me know